### ISSUE TITLE
Crash when entering wrong password twice fast.

### ISSUE THREAD

**Issue by sarg** [prikhi/lightdm-mini-greeter#37]
To reproduce:
- enter wrong password
- while "Incorrect password" message isn't shown enter password second time
- observe lightdm crash

```
** Message: 22:14:53.340: Using entered password to authenticate
** Message: 22:14:54.893: Using entered password to authenticate

** (lightdm-mini-greeter:30916): CRITICAL **: 22:14:54.893: lightdm_greeter_respond: assertion 'priv->n_responses_waiting > 0' failed
```

**Comment by prikhi** [comment]
Thanks, I'll probably disable the input while we're waiting for authentication to complete.

**Comment by prikhi** [comment]
Fixed in 073b6876d418c97d6bd82cb122617cd1434008ca.
